.github/actions/publish-android/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Publish android image'
+
+inputs:
+  abis: 
+    description: 'A space separated list of ABIs to create a AAR for'
+    default: 'armv7 arm64 x86 x86_64'
+  docker-img-name: 
+    description: 'Name of the android image'
+    required: true
+  cache-key:
+    description: 'Cache key under which the android image is stored'
+    required: true
+  rar-path:
+    description: 'Path where the image is stored'
+    default: /tmp/imgcache/img_android.rar
+
+env:
+  DOCKER_BUILDKIT: 1
+
+runs:
+  using: "composite"
+  steps:
+    - name: Build, run android wrapper tests, and publish artifacts
+      run: |
+        set -x
+        # docker run --name test-android-wrapper -v $PWD:/home/indy/libvcx-absa:rw ${{ inputs.docker-img.name }} \
```

Comment:
I suppose this can be removed?

Reply:
Yes, we want to reduce the docker image size by mounting the source as volume instead of copying it in at buildtime, as described in this [issue](https://github.com/AbsaOSS/libvcx/issues/49).